tests/core/selection/fake.js

```diff
@@ -1074,7 +1074,7 @@ bender.test( {
 		editor.setReadOnly( true );
 
 		bot.setData( '<p>[[placeholder]]</p>', function() {
-			var widget = editor.widgets.instances[ 0 ],
+			var widget = bender.tools.objToArray( editor.widgets.instances )[ 0 ],
```

Comment:
The reason of this PR was this issue #1632 where I needed to use new `objToArray` function in my unit tests in the same file to extract widget instances. Because of that this test should depend on `objToArray` function to skip error where tests are not in the correct order and `editor.widgets.instances` have invalid ids.

Reply:
Ok, so as `editor.widgets.instances[ 0 ]` relays on widget id (and widget ids are incremental) if there were some widgets used in previous tests, the widget in this tests would have id greater than 0 thus `editor.widgets.instances[ 0 ]` will return `undefined`.

This is not the case now as test was passing before this change, but it gives the test more flexibility 👍